src/20240220_AaveV3Arbitrum_AaveProtocolEmbassy/AaveProtocolEmbassy.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Aave Protocol Embassy"
+author: "karpatkey_TokenLogic_ACI"
+discussions: "https://governance.aave.com/t/arfc-establishing-the-aave-protocol-embassy-ape/16445"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0xe9da0e50526a98a55aae743f44afc21a86076a12184a6c6c9022aa63dcb0be73"
+---
+
+## Simple Summary
+
+Transfer all available ARB on Aave V3 Arbitrum to the Aave Protocol Embassy multi-sig.
+
+## Motivation
+
+The Aave DAO possesses significant voting power in various governance tokens. However, the current structure of the collector contract hinders direct participation in other DAOs' governance. To effectively leverage this power and actively participate in pivotal governance decisions, such as LTIPs on Arbitrum, the creation of APE is essential.
+
+This will allow the Aave DAO to have a meta-governance embassy represented by Aave DAO service providers and significant delegates.
+
+## Specification
+
+- Transfer all available ARB on the Collector on Arbitrumm to SAFE Multi-sig 0xa9e777D56C0Ad861f6a03967E080e767ad8D39b6
```

Comment:
Is this address mentioned in some post in the forum or something? cant manage to find it anywhere (its basically to put some kind of validation that its the correct safe)

Reply:
added to the AIP and forum: https://governance.aave.com/t/arfc-establishing-the-aave-protocol-embassy-ape/16445/9